src/CrossOrigin.ts

```diff
@@ -1489,14 +1489,10 @@ export class CrossOriginFocusedElementState
         this._transactions = transactions;
     }
 
-    protected dispose() {
+    dispose() {
         super.dispose();
```

Comment:
I suppose we can ditch the methods which only call the superclass here and below.